frontend/src/modules/history/components/Translation.js

```diff
@@ -168,6 +168,78 @@ export class TranslationBase extends React.Component<InternalProps, State> {
         </Localized>
     }
 
+    renderCommentSummary() {
+        const comments = this.props.translation.comments;
+        const commentCount = comments.length;
+
+        if (commentCount === 0) {
+            return null;
+        }
+
+        const authors = [...new Set(comments.map(comment => comment.username))];
+        const lastComment = [...comments].pop();
```

Comment:
There's no need to copy the `comments` array here. You can simply do `const lastComment = comments[commentCount - 1];`.